components/Member.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react"
+import styles from "./members.module.css"
+import Image from "next/image"
+import bannerImgDefault from "../public/blank_banner.png"
+
+export default function Member(props) {
+ 
+  const profile_image = props.profile_image_url.replace("_normal", "")
+  const description = props.description.replace(/\n/g, ' ').trim()
+
+  return (
+    <div className={styles.holdBox}>
```

Comment:
Won't push for tailwind adoption here since the argument is we are a small web app. However, any additional changes outside of this file will require using tailwindcss.